website/docs/software/device/critical-faults.md

```diff
@@ -0,0 +1,20 @@
+---
+id: critical-error-codes
+title: Critical error codes
+sidebar_label: Critical error codes
+---
+
+The device might report these fault codes on the screen, but it will also be outputted on the device serial output. If you encounter a fault code, please post on the forum and we'll try to help.
+
+| Name | Number | Description |
+| ---- | ------ | ----------- |
+| None | 0 |  |
```

Comment:
I'd remove this

Reply:
Removed the "none" line, or do you mean all of those rows?

Reply:
https://meshtastic-git-device-software-apt105.vercel.app/docs/software/device/critical-error-codes